pkg/ccl/logictestccl/tests/local-read-committed/local-read-committed_test: TestReadCommittedLogic_row_level_security failed

pkg/ccl/logictestccl/tests/local-read-committed/local-read-committed_test.TestReadCommittedLogic_row_level_security [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18792557?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18792557?buildTab=artifacts#/) on master @ [d7dbf95c7979d2d31c3a44f4688f248bbfc5f223](https://github.com/cockroachdb/cockroach/commits/d7dbf95c7979d2d31c3a44f4688f248bbfc5f223):


```
=== RUN   TestReadCommittedLogic_row_level_security
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/dc698855159f6c9da6a4e70dc8a23d0a/logTestReadCommittedLogic_row_level_security1292514296
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:141: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
[21:17:41] setting distsql_workmem='84122B';
    logic.go:4452: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/dc698855159f6c9da6a4e70dc8a23d0a/logTestReadCommittedLogic_row_level_security1292514296
--- FAIL: TestReadCommittedLogic_row_level_security (1.31s)
=== RUN   TestReadCommittedLogic_row_level_security/create_drop_sanity
    logic.go:3077: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30775/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/local-read-committed/local-read-committed_test_/local-read-committed_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/row_level_security:76: select "eventType" from system.eventlog order by timestamp desc limit 1;
        expected:
            drop_policy
            
        but found (query options: "") :
            create_policy
            
    logic.go:2288: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30775/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/local-read-committed/local-read-committed_test_/local-read-committed_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/row_level_security:81: too many errors encountered, skipping the rest of the input
[21:17:42] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30775/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/local-read-committed/local-read-committed_test_/local-read-committed_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/row_level_security with config local-read-committed: 20 tests, 2 failures
--- total: 20 tests, 2 failures
    --- FAIL: TestReadCommittedLogic_row_level_security/create_drop_sanity (0.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReadCommittedLogic_row_level_security.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-47559

--- a/pkg/sql/logictest/testdata/logic_test/row_level_security
+++ b/pkg/sql/logictest/testdata/logic_test/row_level_security
@@ -48,7 +48,7 @@ statement ok
 CREATE POLICY p1 on sanity1 USING (true);
 
 # Verify we log to the event log.
-query T
+query T retry
 select "eventType" from system.eventlog order by timestamp desc limit 1;
 ----
 create_policy
@@ -72,7 +72,7 @@ statement ok
 DROP POLICY p1 on sanity1;
 
 # Verify we log to the event log.
-query T
+query T retry
 select "eventType" from system.eventlog order by timestamp desc limit 1;
 ----
 drop_policy

